### PR TITLE
Geoserver sharing layers

### PR DESCRIPTION
**Pull request by richardforsterNVBW**
# Changes

## MobiData-BW:sharing_vehicles

* added `station_id` to `MobiData-BW:sharing_vehicles`
* added cql filter to only include vehicles where `station_id is null`

## MobiData-BW:sharing_stations

* added new layer "Keine Echtzeitdaten" to `mdbw_sharing_stations_default`

### CODE REVIEW

**Comment by richardforsterNVBW** [comment]
@hbruch: Can you please check, if all the changes are correct? Afterwards, I can still squash the commits. 

I interprete the `station_id` als `nillable`. This is why I set it to `true`. Was `nillable` recently added? I don't remember it being part of the `featuretype.xml` the last time I did a commit.

Since it is not a mandatory attribute in the `MobiData-BW:sharing_vehicles` layer, `minOccurs` is 0 and `maxOccurs` is 1.

File: etc/geoserver/workspaces/MobiData-BW/styles/mdbw_sharing_stations_default.sld [hunk 1]
```diff
@@ -1,21 +1,64 @@
...
-              <ogc:Literal>0</ogc:Literal>
-            </ogc:PropertyIsGreaterThan>
+            <ogc:And>
+              <ogc:Not>
+                <ogc:Or>
+                  <ogc:PropertyIsLike wildCard="*" singleChar="#" escape="!">
+                    <ogc:PropertyName>feed_id</ogc:PropertyName>
+                    <ogc:Literal>stadtmobil*</ogc:Literal>
+                  </ogc:PropertyIsLike>
+                  <ogc:PropertyIsEqualTo>
```
Comment (hbruch) [inline etc/geoserver/workspaces/MobiData-BW/styles/mdbw_sharing_stations_default.sld:51]: Shouldn't flinkster be listed here as well?

**Review by hbruch — APPROVED** [review]
LGTM. Please consider adding flinkster as having no realtime data, also.

File: etc/geoserver/workspaces/MobiData-BW/styles/mdbw_sharing_stations_default.sld (hunk 1 above)
Comment (richardforsterNVBW) [reply etc/geoserver/workspaces/MobiData-BW/styles/mdbw_sharing_stations_default.sld:51]: According to @ThorstenFroehlinghaus, Flinkster has realtime data.

Comment (richardforsterNVBW) [reply etc/geoserver/workspaces/MobiData-BW/styles/mdbw_sharing_stations_default.sld:51]: For now, I won't include this feed until it was clarified if it is realtime or not.

**Pull request merged**